core/src/test/resources/org/bitcoinj/script/script_invalid.json

```diff
@@ -151,8 +151,8 @@
 ["2 2 LSHIFT", "8 EQUAL", "P2SH,STRICTENC", "disabled"],
 ["2 1 RSHIFT", "1 EQUAL", "P2SH,STRICTENC", "disabled"],
 
-["1","NOP1 NOP2 NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 2 EQUAL", "P2SH,STRICTENC"],
```

Comment:
After I have been spending several hours comparing bitcoinj with bitcoin-core, and I concluded that the script evaluation should be refactored a little bit to cover the same tests.

I think it's independent from this PR, isn't it? I could work on filling the gap with bitcoin-core, considering it's quite important for the work I'm doing. 

Let me know how to proceed. Your repo, your rules :)